pymetamap/SubprocessBackend.py

```diff
@@ -68,19 +68,19 @@ def extract_concepts(self, sentences=None, ids=None,
 
         input_file = None
         if sentences is not None:
-            input_file = tempfile.NamedTemporaryFile(delete=False)
+            input_file = tempfile.NamedTemporaryFile(mode="wb", delete=False)
         else:
-            input_file = open(filename, 'r')
-        output_file = tempfile.NamedTemporaryFile(delete=False)
+            input_file = open(filename, 'wb')
```

Comment:
I updated the file using github's online editor and created another commit to fix the input file issue.